service/collector/bridge.go

```diff
@@ -0,0 +1,164 @@
+package collector
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"regexp"
+
+	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/prometheus/client_golang/prometheus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	subsystemBridge = "bridge"
+)
+
+var (
+	bridgeClusterWithoutNetworkInterfaceDesc *prometheus.Desc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, subsystemBridge, "cluster_without_network_interface"),
+		"Clusters without network interface.",
+		[]string{
+			labelCluster,
+		},
+		nil,
+	)
+	bridgeNetworkInterfaceWithoutClusterDesc *prometheus.Desc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, subsystemBridge, "network_interface_without_cluster"),
+		"Network interfaces without cluster.",
+		[]string{
+			labelCluster,
+		},
+		nil,
+	)
+)
+
+type BridgeConfig struct {
+	G8sClient versioned.Interface
+	Logger    micrologger.Logger
+}
+
+type Bridge struct {
+	g8sClient versioned.Interface
+	logger    micrologger.Logger
+}
+
+func NewBridge(config BridgeConfig) (*Bridge, error) {
```

Comment:
Here is the new collector implementation. It is pretty similar to the existing one.